src/ModuleList/Value/Value.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+
+import styled from '@emotion/styled';
+
+import { useValueMeta } from '../../common/api/curriculum/value';
+
+interface Params {
+  valueId: string;
+};
+
+interface Value {
+  value: string;
+  inspiration: {
+    organization: string;
+    definition: string;
+  }[];
+}
+
+const StyledReflection = styled.h1`
+  font-size: 25px;
+  color: white;
+  font-weight: bold;
+  margin: 1.25rem auto;
+`;
+
+const StyledBox = styled.div`
+  width: auto;
+  border: 2px solid white;
+  padding: 20px;
+  margin: 1.25rem auto;
+  font-style: italic;
+
+  h2 {
+    text-align: center;
+    color: white;
+    font-weight: bold;
+  }
+
+  p {
+    color: white;
+    text-align: center;
+  }
+`;
+
+const StyledTextArea = styled.textarea`
+  background-color: #bbd3ea;
+  width: 100%;
+  height: 150px;
+  border: 2px solid white;
+  padding: 10px;
+  margin: 1.25rem auto;
+
+
+  ::placeholder {
+    color: black;
+  }
+`;
+
+const StyledSavebutton = styled.button`
+  background-color: #6ca2d8;
+  border: 2px solid white;
+  color: white;
+  padding: 10px 32px;
+  text-align: center;
+  font-size: 16px;
+  margin: 1.25rem auto;
+  border-radius: 10px;
+
+  float: right;
+`;
```

Comment:
Can you make a styled.ts file and place these styled components there?  Then import them back into the Value component.